modules/ega_download/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+
+All notable changes to the `ega_download` module will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0] - 2021-10-06
+
+This release was authored by Kostiantyn Dreval.
+
+- The input is a sample table of the following example:
```

Comment:
I love this example table here. Is it worth putting a note in the default.yaml that this example is here in the changelog? I don't think users would think to look here for an example. The description in the default.yaml seems okay as it is though, if you don't want to mention this example table.